src/main/java/org/spongepowered/api/world/portal/PortalTypes.java

```diff
@@ -22,19 +22,20 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.spongepowered.api.world.teleport;
+package org.spongepowered.api.world.portal;
 
 import org.spongepowered.api.Sponge;
 
 import java.util.function.Supplier;
 
-public final class PortalAgentTypes {
+public final class PortalTypes {
 
-    // The default agent used to handle traveling to nether and the end
-    public static final Supplier<PortalAgentType> DEFAULT = Sponge.getRegistry().getCatalogRegistry().provideSupplier(PortalAgentType.class, "DEFAULT");
+    public static final Supplier<PortalType> NETHER = Sponge.getRegistry().getCatalogRegistry().provideSupplier(PortalType.class, "NETHER");
```

Comment:
`THE_END` and `THE_NETHER` to be consistent with dimension and generator types naming?

Reply:
I don't think that makes sense here. We generate "the nether" and "the end" (there is one nether dimension type and one nether generator, for example) - but we use "nether" and "end" portals, not "the nether" or "the end" portals - of which there may be multiple. I think it's correct as is.